packages/manager/apps/public-cloud/src/assets/theme/_stepper.less

```diff
@@ -0,0 +1,8 @@
+@import "~ovh-ui-kit/packages/oui-responsive/_variables";
+
+@media screen and (max-width: @oui-responsive-breakpoint-tablet) {
+  oui-step-form { // stylelint-disable-line
+    padding-left: 0 !important;
```

Comment:
No way to increase the specificity instead of using `!important` ?

For example,
```css
oui-step oui-step-form { // stylelint-disable-line
    padding-left: 0;
    padding-right: 0;
}
```

This would make the specificity go from (0,1,0) to (0,2,0)

Or add an id to the stepper:
```css
#myStepper oui-step-form { // stylelint-disable-line
    padding-left: 0;
    padding-right: 0;
}
```

would make the specificity (1,1,0), it should be enough